entities/position_test.go

```diff
@@ -28,8 +28,8 @@ func initPool() (*Pool, int, int) {
 		panic(err)
 	}
 
-	tickSpacing := constants.TickSpacings[constants.FeeLow]
-	p, err := NewPool(DAI, USDC, constants.FeeLow, poolSqrtRatioStart, big.NewInt(0), big.NewInt(0), poolTickCurrent, nil)
+	tickSpacing := constants.TickSpacings[constants.FeeMedium]
```

Comment:
just for understanding, why do we have to change the fee in this test?

Reply:
Because I need to test tickSpacing. FeeLow has been changed from 10 to 4 and now is 1:
- tick lower must be multiple of tick spacing
- tick upper must be multiple of tick spacing